hackathon_site/dashboard/frontend/src/pages/Teams/Teams.module.scss

```diff
@@ -0,0 +1,7 @@
+@import "./../../assets/abstracts/mixins";
+@import "./../../assets/abstracts/variables";
+
+.teamsDashboard {
```

Comment:
This is very picky but could you rename this to `teamsListGridItem` because teamsDashboard seems like this style is meant for the entire page but it's not (css classnames should be clear & user should know exactly what it's styling at a glance)

Reply:
done!